lib/matplotlib/contour.py

```diff
@@ -757,44 +792,6 @@ class ContourSet(cm.ScalarMappable, ContourLabeler):
     """
 
     def __init__(self, ax, *args, **kwargs):
-        """
```

Comment:
numpydoc says to put the parameters section in the class docstring (not the ``__init__`` docstring, before the attributes doc, which I think makes sense, and makes everything look a bit more sensible.